tests/journey/test_federal_senate_dataset.py

```diff
@@ -0,0 +1,37 @@
+import os
+from tempfile import gettempdir
+from unittest import main, skipIf, TestCase
+
+
```

Comment:
Also on the matter of imports: `from unittest import TestCase, main, skipIf` and there is an extra line here that should bem removed